projects/ad7768-evb/Makefile

```diff
@@ -0,0 +1,10 @@
+# Uncomment to use the desired platform
+PLATFORM = xilinx
```

Comment:
This only works for xilinx? Otherwise you can comment this. Anyway if not specified, xilinx or altera will be used depending on the hdf file.

Reply:
Will comment this.